src/app/features/admin/admin.module.ts

```diff
@@ -1,28 +1,19 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+
 import { AdminRoutingModule } from './admin-routing.module';
-import { CategoryListComponent } from './components/category-list/category-list.component';
-import { ProductListComponent } from './components/product-list/product-list.component';
-import { OrderListComponent } from './components/order-list/order-list.component';
-import { AddCategoryComponent } from './components/add-category/add-category.component';
-import { AddProductComponent } from './components/add-product/add-product.component';
 import { SharedModule } from 'src/app/shared/shared.module';
-
-
+import { CategoryService } from './services/category.service';
+import { ProductService } from './services/product.service';
+import { AdminComponent } from './admin.component';
 
 @NgModule({
-  declarations: [
-    CategoryListComponent,
-    ProductListComponent,
-    OrderListComponent,
-    AddCategoryComponent,
-    AddProductComponent,
-  ],
+  declarations: [AdminComponent],
   imports: [
     CommonModule,
     AdminRoutingModule,
     SharedModule,
```

Comment:
You don't have to import this module here because you don't use any of the shared logic here